### DISCUSSION TITLE
Switching from dbt cloud CLI to dbt core

### DISCUSSION THREAD

**Discussion by huntleyls** [dagster-io/dagster#22560] [Q&A]
Hi, 
I have a project and it still points to dbt cloud packages not dbt core. I've uninstalled it and when i fire up dagster dev and check what dependencies its still dbt cloud any suggestions?

**Comment by garethbrickman** [comment]
Are you using a virtual environment? If so may be worth recreating a fresh environment to ensure no traces of dbt Cloud dependencies. 

Otherwise sometimes old dependencies might be cached. Clear any caches that might be holding onto the old dbt Cloud packages via the [pip cache remove/purge](https://pip.pypa.io/en/stable/cli/pip_cache/) commands.

**Reply by huntleyls** [reply]
i created an enviroment and then cleared all instances of it on my machine it still doesnt work

**Reply by garethbrickman** [reply]
Could you try aliasing `dbt` command so that ​dbt​ is reserved for the dbt Core CLI, as recommended in https://github.com/dagster-io/dagster/discussions/20205#discussioncomment-9795736

**Reply by huntleyls** [reply]
I did the aliasing and i still get 

dbt version: dbt Cloud CLI - 0.37.20 (8e56efc8c1d393486464adc8602f86bd466166b0 2024-06-14T02:30:06Z)

**Reply by garethbrickman** [reply]
Are you sure you're operating within a virtual environment? Could you share what your terminal looks like when doing this?

**Reply by allen-morales** [reply]
I am encountering the same problem.

**Reply by kamidzi** [reply]
I am also encountering this.
